metricbeat/etc/beat.yml

```diff
@@ -55,4 +55,5 @@ metricbeat.modules:
   period: 2s
   processes: ['.*']
 
-
+  # if true, exports the CPU usage in ticks, together with the percentage values
+  cpu_usage_in_ticks: false
```

Comment:
Sorry, I missed it. Maybe we should put config.yml directly in the module/<module-name> directory, so it's more visible to the users if it's so important.